spec/migrations/20150921204114_add_vmware_ro_datastores_to_hosts_storages_spec.rb

```diff
@@ -2,11 +2,38 @@
 require_migration
 
 describe AddVmwareRoDatastoresToHostsStorages do
+  class AddVmwareRoDatastoresToHostsStorages::HostsStorage < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+    self.table_name = "hosts_storages"
+  end
+
+  let(:host_storages_stub) { migration_stub(:HostStorage) }
   let(:hosts_storages_stub) { migration_stub(:HostsStorage) }
 
   migration_context :up do
     it "Adds ID in correct region" do
+      seq_start = ActiveRecord::Base.rails_sequence_start
+      seq_start = 1 if seq_start == 0
+
+      h = hosts_storages_stub.create(:host_id => 1, :storage_id => 2)
+      migrate
+      h = host_storages_stub.first
+
+      expect(h.id).to eq(seq_start)
```

Comment:
It looks like in_current_region? just checks if `region_number == self.class.my_region_number`, but I can check if `id_to_region(h.id) == my_region_number()`